c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -1041,20 +1041,22 @@ mfxStatus MfxC2DecoderComponent::HandleFormatChange()
     bool need_realloc = false;
     mfx_res = m_mfxDecoder->DecodeHeader(m_c2Bitstream->GetFrameConstructor()->GetMfxBitstream().get(), &m_mfxVideoParams);
     MFX_DEBUG_TRACE__mfxStatus(mfx_res);
-    if (MFX_ERR_NONE == mfx_res) {
+    // if (MFX_ERR_NONE == mfx_res) {
         // Query required surfaces number for decoder
-        mfxFrameAllocRequest decRequest = {};
-        mfx_res = m_mfxDecoder->QueryIOSurf(&m_mfxVideoParams, &decRequest);
-        if (MFX_ERR_NONE == mfx_res) {
-            if (m_mfxVideoParams.mfx.FrameInfo.Width != m_uMaxWidth ||
-                m_mfxVideoParams.mfx.FrameInfo.Height != m_uMaxHeight) {
-                need_realloc = true;
-            } else if (m_uOutputDelay < decRequest.NumFrameSuggested){
+        // mfxFrameAllocRequest decRequest = {};
+        // mfx_res = m_mfxDecoder->QueryIOSurf(&m_mfxVideoParams, &decRequest);
+        // if (MFX_ERR_NONE == mfx_res) {
+            // if (m_mfxVideoParams.mfx.FrameInfo.Width != m_uMaxWidth ||
+            //     m_mfxVideoParams.mfx.FrameInfo.Height != m_uMaxHeight) {
+            //     need_realloc = true;
+            // } else if (m_uOutputDelay < decRequest.NumFrameSuggested){
```

Comment:
I would suggest not use // to mask block code, please use #if 0 if this code may add back later, or remove them directly.

Reply:
ok, I'll detele it.